src/reynir/verbframe.py

```diff
@@ -322,17 +328,19 @@ def create_from_config(cls, s: str) -> None:
         if error:
             # Add this to the error database
             VerbErrors.add_error(verb, args, prepositions, particle, error)
-        else:
-            # Create a VerbFrame instance
-            vf = cls(verb, args, prepositions, particle, score)
-            case_key = vf.case_key
-            if case_key is not None:
-                # This verb frame has cases as arguments
-                cls.CASE_FRAMES[case_key].append(vf)
-            # Add to the dictionary of all verb frames
-            cls.ALL_FRAMES[vf.key].append(vf)
-            # Add to the set of known verb lemmas
-            cls.VERBS.add(verb)
+        # Create a VerbFrame instance
+        # Note: In order to parse verbs with wrong arguments,
+        # the frame needs to be present as a regular VerbFrame instance
+        # that is then marked as an error in GreynirCorrect
+        vf = cls(verb, args, prepositions, particle, score)
```

Comment:
We need to discuss this - GreynirPackage should not allow a parse with a wrong argument, but GreynirCorrect should. This is normally effected by having the latter override standard behavior in the former, e.g. via a custom parser class (like GreynirCorrect implements).

Reply:
We have ruminated on this before but hadn't reached a conclusion. Allowing known, wrong structures (with a severe score reduction) would make GreynirPackage more robust, and could help unparsable sentences. 
As we have another solution up our sleeve, I'll look into how the error grammar rules are excluded except in GreynirCorrect and try to imitate that.